mmhuman3d/data/data_converters/humman.py

```diff
@@ -0,0 +1,277 @@
+import glob
+import os
+
+import numpy as np
+from tqdm import tqdm
+
+from mmhuman3d.core.conventions.keypoints_mapping import (
+    convert_kps,
+    get_keypoint_idx,
+    get_keypoint_num,
+)
+from mmhuman3d.data.data_structures import SMCReader
+from mmhuman3d.data.data_structures.human_data import HumanData
+from .base_converter import BaseModeConverter
+from .builder import DATA_CONVERTERS
+
+
+@DATA_CONVERTERS.register_module()
+class HuMManConverter(BaseModeConverter):
+    """A mysterious dataset that will be announced soon."""
+
+    skip_no_iphone = True
+    skip_no_keypoints3d = True
+    downsample_ratio = 10  # uniformly select 1 from every 10 examples
```

Comment:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/73/commits/f943649f4a1fd45aa0e9e7134989f0b4c127b31e.

Reply:
Previously I thought it may be more flexible for user to pass args when instantiate HuMManConverter. In some cases, they may want to change some configs, e.g., `downsample_ratio`, without modifying the code.

As only we use it currently, it can be left for future discussion.